monasca_agent/collector/checks_d/ceph.py

```diff
@@ -51,6 +52,10 @@ def check(self, instance):
         self.CLUSTER = instance.get('cluster_name', 'ceph')
         self.dimensions = self._set_dimensions({'ceph_cluster': self.CLUSTER,
                                                 'service': 'ceph'}, instance)
+        match_version = re.search(
+                            _CEPH_VERSION_REGEX,
+                            self._ceph_cmd('version'))
+        self.preluminous = match_version.group(1).split('.') < ['12','2','0']
```

Comment:
How about using something like the snippet below to better describe what '12','2','0' is, and also make the version parsing more robust?

```
>> from pkg_resources import parse_version
>> _CEPH_LUMINOUS_RELEASE_VERSION = '12.2.0'
>> parse_version('12.2.1.dev') > parse_version(_CEPH_LUMINOUS_RELEASE_VERSION)
>> True
```